### ISSUE TITLE
sass task cannot complete

### ISSUE THREAD

**Issue by cezarneaga** [braginteractive/mdlpress#1]
An error occured: EISDIR: illegal operation on a directory, read

i did this and same thing happens on both your repository and my starter theme. 
any ideas?

PS: why don't you install MDL from NPM too?

**Comment by braginteractive** [comment]
Any reason why you are using this theme over MDLWP?

Theme uses Bower to install MDL, no reason other than I like using it..

**Comment by braginteractive** [comment]
I have edited the gulpfile.js in MDLWP to fix this issue. Check it out here: https://github.com/braginteractive/MDLWP/blob/master/gulpfile.js

**Comment by cezarneaga** [comment]
i am doing the course. i saw the gulp file. doesnt it need the 

```
includePaths: [
    config.bowerDir + 'material-design-lite/src'
]
```

part anymore?

**Comment by braginteractive** [comment]
Yes it will need that part to pull in the Sass files. I forgot MDLWP doesn't use the Sass files. If you want to practice and submit a Pull Request once you get it working, I will merge your changes.

**Comment by cezarneaga** [comment]
I am doing this during free evenings. might take a while. :)

**Comment by Marshallsbest** [comment]
Hey Brad I have added those sasss options and got that runnning gulp is working as well, Only I'm running windows 10 and the little notification doesnt work ( no matter) . How ever I am now  having trouble with your walker class. I can't get the reconfigured menu to display, i was wondering if this is possibly outof date with the recent word press up dates?

**Comment by braginteractive** [comment]
What is the issue you are having?

**Comment by Marshallsbest** [comment]
I'm still learning and I decided to take another run at doing my own theme. (Ive since decided  to modify DIVI due to time constraints) I got to the  Walker class portion of your tutorial and it wouldnt create the menu for me, you can link it here git@github.com:Marshallsbest/_marshallsbest.git  if you want to have  a look, I also  added the fix for the gulp file that worked for me as well as the way SASS has to addresss the MDL folder in the style.scss. Great work thus far keep it up your the closest to google MDL that I've seen.
